R/zed-scores.R

```diff
@@ -0,0 +1,90 @@
+# z-score calculations from Doug's tables pdf. This will soon be deprecated and
+# replaced using the cross product of the margins for expected values.
+zScoresDep <- function (table, margin) {
+    other_margin <- 3 - margin ## Assumes 2-D
+
+    sample_prop <- prop.table(table, margin)
+    # doug's C (for rows)
+    off_margin <- prop.table(margin.table(table, other_margin))
+
+    # adjusted (for empty cells), unweighted counts for the denom
+    adjusted_table <- bases(table, c(1, 2)) + 1
```

Comment:
Since you're calling `bases`, does that mean that this function is only intended for use on `CrunchCubes`? Do we need to `stopifnot` check the input class?